radixconfig.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: radix.equinor.com/v1
+kind: RadixApplication
+metadata:
+  name: procosys-library-api
+spec:
+  environments:
+  - name: dev
+  - name: test
+  - name: prod
+
+  components:
+  # ---------
+  # Backend
+  #---------- 
+
+  - name: backend
+    image: procosys.azurecr.io/library/api:{imageTagName}
+    ports:
+    - name: http
+      port: 5000
+    publicPort: http
+    secrets:
+    - CONNECTIONSTRINGS__APPCONFIG
+    environmentConfig:
+    # Development environment
+    - environment: dev
+      identity:
+        azure:
+          clientId: fdf5086c-009f-4407-b7bb-d2ece26dea45
+      imageTagName: latest
+      variables:
+        ASPNETCORE_ENVIRONMENT: 'Development'
+        UseAzureAppConfiguration: 'true'
+      monitoring: true
+      resources:
```

Comment:
Is the size of memory/cpu resources carefully considered for library,k or just copied from ipo or preservation setup?
I ask because I guess library is a very lightweight app as it is just now (thin "proxy" without DB)

Reply:
We have found that we use max 177mb. Request is set to 256, so we will probably not use more than that. Limit is set to 1024 to prevent the system for crashing. But I think we can limit to 256 also, because if we use more memory there must be something wrong.